src/index.ts

```diff
@@ -3,6 +3,14 @@ import { propsShorthandRule } from "./rules/props-shorthand";
 import { requireSpecificComponentRule } from "./rules/require-specific-component";
 
 export = {
+  configs: {
+    plugins: ["chakra-ui"],
+    recommended: {
+      "chakra-ui/props-order": "error",
+      "chakra-ui/props-shorthand": "error",
+      "chakra-ui/require-specific-component": "error",
+    },
```

Comment:
It may be unnecessary, but I added it here because I deleted rule.docs.recommended.